internal/glfwwin/wglcontext_windows.go

```diff
@@ -378,8 +378,8 @@ func initWGL() error {
 	prc := wglGetCurrentContext()
 
 	if err := wglMakeCurrent(dc, rc); err != nil {
-		wglMakeCurrent(pdc, prc)
-		wglDeleteContext(rc)
+		_ = wglMakeCurrent(pdc, prc)
+		_ = wglDeleteContext(rc)
```

Comment:
These should return the error

Reply:
```go
	if err := wglMakeCurrent(dc, rc); err != nil {
		if err := = wglMakeCurrent(pdc, prc); err != nil {
			return err
		}
		if err := wglDeleteContext(rc); err != nil {
			return err
		}
		return err
	}
```
like so?

Reply:
Ah, ok in this context the error already happened. Let's ignore it. Sorry for going back and forth.